site/docs/guides/admin/servers/configuring-the-view-services.md

```diff
@@ -209,19 +209,19 @@ It is possible to list the configured view services for an [OMAG Server](/concep
                 "serviceName": "Glossary Author",
                 "serviceURLMarker": "glossary-author",
                 "serviceDescription": "View Service for glossary authoring.",
-                "serviceWiki": "https://odpi.github.io/egeria/open-metadata-implementation/access-services/subject-area/"
+                "serviceWiki": "https://odpi.github.io/egeria-docs/services/omvs/glossary-author/overview/"
             },
             {
                 "serviceName": "Repository Explorer",
                 "serviceURLMarker": "rex",
                 "serviceDescription": "Explore open metadata instances.",
-                "serviceWiki": "https://odpi.github.io/egeria/open-metadata-implementation/view-services/rex-view/"
+                "serviceWiki": "https://odpi.github.io/egeria-docs/services/omvs/rex/overview/"
             },
             {
                 "serviceName": "Type Explorer",
                 "serviceURLMarker": "tex",
                 "serviceDescription": "Explore the open metadata types.",
-                "serviceWiki": "https://odpi.github.io/egeria/open-metadata-implementation/view-services/tex-view/"
+                "serviceWiki": "https://odpi.github.io/egeria-docs/services/omvs/tex/overview"
```

Comment:
Think about this, this is code returned in a response to an Egeria API. We should correct the Egeria code first before changing the docs to match the corrected value. to you fancy doing this? In the mean time I would remove these changes from this pr and raise another issue to track this in the docs and egeria core repo.

Reply:
Yeah I can do that for these as well - was doing the same for `site/docs/guides/diagnostic/integration-daemon-diagnostic-guide.md` and `site/docs/guides/contributor/documentation.md` so happy to do it here also. I think [this comment](https://github.com/odpi/egeria-docs/pull/339#discussion_r812074892) just confused me with regards this specific example. Will remove these changes now.